server/src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -366,7 +366,7 @@ private void cleanupAndDelete(Owner owner, boolean revokeCerts) {
             }
             else {
                 // otherwise just remove them without touching the CRL
-                poolManager.removeAllEntitlements(c);
+                poolManager.revokeAllEntitlements(c);
```

Comment:
@nguyenfilip I'm in favor of the second approach, simply cause if there are some use cases where we do not need to compute compliance ( or delay it ), then there might be some advantage in skipping it.